### PR TITLE
fix: passes pact_dir to publish()

### PR DESCRIPTION
**Pull request by anneschuth**
MessagePact was not passing the pact_dir to the publish() method. This caused tests to fail.

### CODE REVIEW

**Comment by anneschuth** [comment]
This may be a duplicate of #209

**Comment by elliottmurray** [comment]
Ha yes it may be! Thanks for the contribution but do you want to check if the latest version resolves your issue. If not I'll look closer at this (I didn't test it specifically for messages but in th e2e example)

File: pact/__version__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """Pact version info."""
 
-__version__ = '1.3.2'
+__version__ = '1.3.3'
```
[the remark this answers is not in the document]
Comment (anneschuth) [reply pact/__version__.py:3]: Right, I'll drop this from the PR.

**Comment by anneschuth** [comment]
I was running 1.3.2, that #209 was part of and was still running into the issue. This PR fixes it for me.

**Comment by elliottmurray** [comment]
ok it  LGTM - I'll test properly tonight and try and get a release done.

**Pull request merged**